### DISCUSSION TITLE
Devcontainer database

### DISCUSSION THREAD

**Discussion by mojmirdurik** [inventree/InvenTree#6197] [Q&A] [answered]
Hello,

I would like to ask:

- Which database is used (SQLite, PostgreSQL, ...) when project is launched in devcontainer?
- If PostgreSQL is used how can I access it using pgAdmin or adminer or psql?

Thank you in advance, Mojo :)

**Comment by wolflu05** [comment, marked as answer]
Hey,

- The devcontainer uses sqlite as database, you can see the configuration for it [here](https://github.com/inventree/InvenTree/blob/master/.devcontainer/devcontainer.json#L67-L78)
- If you want to access the database contents, you could use any dbviewer e.g. dbeaver (opensource), datagrip (jetbrains) or even some vscode extensions can handle simple operations.
